### DISCUSSION TITLE
a small 'hole' in the DSGVO compliance of bs cookies settings.

### DISCUSSION THREAD

**Discussion by kreincke** [bootscore/bootscore#598] [Ideas]
Dear bs-Team;

I've just described the integration of bs-Cookie-Settings into my site. One post deals with bs-Cookie-Settings as dialog (https://fodina.de/properly-used-cookies/, and the other with the respective evaluation (https://fodina.de/properly-managed-cookies/). In sum, everything works well.

Nevertheless, there seems to be a (very) little hole in the GPDR compliance: At least in Europe, website designers are requested to ask their readers for permission before they save any cookies to the readers' computers. Moreover: they have to inform their readers which cookies they will write before they do so.  

The configuration dialog of bs-cookies-settings allows the website designers to use RegExs for listing the cookies about they want to inform their readers. But, at the starting point, there should not have been written cookies - if they implement their writing function in accordance with the law. So, the lists created by the RegExs are empty and do not serve information to the readers.

Thus, the users of bs-Cookie-Settings should not use the regex functionality if they want to be compliant.

However, a great job, your plugin.

heartfully KR

**Comment by crftwrk** [comment]
You're welcome 👍

https://fodina.de/properly-used-cookies/

> By default, the dialog is placed transparently over the existing text. This prevents our visitors from properly reading the one or the other. We can visually design the consent dialog by adding the following lines to our file _bscore_custom.scss:

Plugin uses `bg-body-tertiary` which is a new color class introduced in Bootstrap 5.3 https://getbootstrap.com/docs/5.3/customize/color/. You are still on v5.2, that's why this color is missed in your theme and bg colors are missed in Modal as well. You should update bootScore theme to tatest v5.3.x https://github.com/bootscore/bootscore/releases.
